cfgov/unprocessed/css/molecules/global-search.less

```diff
@@ -144,80 +152,67 @@
         }
     }
 
-    // Tablet size and below.
+    // Up to desktops
     .respond-to-max( @bp-sm-max, {
-        // Absolute position when there is a flyout at mobile/tablet sizes.
-        // Make the height large so that the overflowing flyout can be hidden,
-        // but not get cropped when it's visible.
-        height: 500px;
-        width: 100%;
-        position: absolute;
-        top: 0;
-        overflow-x: hidden;
-        // TODO: This will be ignored in IE8-10,
-        //       leading to the overflow preventing clicks on the links.
-        pointer-events: none;
```

Comment:
The mouse events shouldn't matter, we're no longer creating a container that covers the entire screen. I'll get IE9 fixed